src/apiClient.ts

```diff
@@ -18,18 +23,32 @@ export interface IAPIClient {
     sendEventToServer: (event: SDKEvent, _options?: Dictionary<any>) => void;
     sendSingleEventToServer: (event: SDKEvent) => void;
     sendBatchForwardingStatsToServer: (
-        forwardingStatsData: ForwardingStatsData,
+        forwardingStatsData: IForwardingStatsData,
         xhr: XMLHttpRequest
     ) => void;
     sendSingleForwardingStatsToServer: (
-        forwardingStatsData: ForwardingStatsData
+        sendSingleForwardingStatsToServer: IForwardingStatsData
     ) => void;
     prepareForwardingStats: (
         forwarder: MPForwarder,
         event: IUploadObject
     ) => void;
 }
 
+export interface IForwardingStatsData {
+    mid: number;
+    esid: number;
+    n: string;
+    attrs: Dictionary<any>;
```

Comment:
Let's avoid using `any`. Either we should make a type for this attribute, or use `Record<string, string>` if it's a simple key/value pair. I would prefer the former if there are a specific attributes that must be passed.